back/engines/commercial/project_folders/app/policies/project_folders/folder_policy.rb

```diff
@@ -24,7 +24,10 @@ def published_folders
     end
 
     def show?
-      true
+      return true if user && UserRoleService.new.can_moderate?(record, user)
```

Comment:
What about `if admin?` ?

I answered my own question. It's [here](https://github.com/CitizenLabDotCo/citizenlab/blob/fd87f1dfdb1595d54b33a0c9c88ef4596515b6d9/back/engines/commercial/project_folders/app/services/project_folders/patches/user_role_service.rb#L6).